.gitignore

```diff
@@ -1,7 +1,6 @@
 *.iml
-.gradle
-/local.properties
-/.idea
-.DS_Store
-/build
-/captures
+.gradle/
+local.properties
+.idea/
+app/build/
```

Comment:
Why removing `.gitignore` in `app` module and adding `app/build/` here? With the old style, we needn't modify anything in this `.gitignore` if I change the module name.

Reply:
hmm alright (but then what kind of thing would require changing the module name?)